examples/cross_decomposition/plot_pcr_vs_pls.py

```diff
@@ -0,0 +1,110 @@
+"""
+==================================================================
+Principal Component Regression vs Partial Least Squares Regression
+==================================================================
+
+This example compares Principal Component Regression (PCR) and Partial Least
+Squares Regression (PLS) on a toy dataset. Our goal is to illustrate how PLS
+can outperform PCR when the target is strongly correlated with some features
+that have a low variance.
+
+PCR simply consists in applying :class:`~sklearn.decomposition.PCA` to the
```

Comment:
```suggestion
PCR simply consists of applying :class:`~sklearn.decomposition.PCA` to the
```

Reply:
Not a native speaker, but consist in and consist of have different meaning and I think this is the correct one https://english.stackexchange.com/questions/61600/consist-in-vs-consist-of

Reply:
Learnt something new, thanks

Reply:
actually i might just be wrong on this one so nevermind. I updated the sentence